tests/umock_c_reals_int/umock_c_reals_int.c

```diff
@@ -3,20 +3,19 @@
 
 
 #ifdef __cplusplus
-#include <cstdint>
```

Comment:
Just for my knowledge, should these included have been caught with the windows build of IWYU?

Reply:
Great question, but I have no idea. The removal made sense to me but I don't know why it wasn't flagged on Windows

---
In reply to: [563862621](https://github.com/Azure/umock-c/pull/187#discussion_r563862621) [](ancestors = 563862621)